bedrock/mozorg/views.py

```diff
@@ -251,20 +251,8 @@ def home_tweets(locale):
     return []
 
 
-def home(request, template='mozorg/home/home.html'):
-    locale = l10n_utils.get_locale(request)
-
-    if lang_file_is_active('mozorg/home/index-2016', locale):
-        template = 'mozorg/home/home.html'
-    else:
-        template = 'mozorg/home/home-voices.html'
-
-    return l10n_utils.render(
-        request, template, {
-            'has_contribute': lang_file_is_active('mozorg/contribute'),
-            'tweets': home_tweets(locale),
-            'mobilizer_link': settings.MOBILIZER_LOCALE_LINK.get(
-                locale, settings.MOBILIZER_LOCALE_LINK['en-US'])})
+def home(request):
+    return l10n_utils.render(request, 'mozorg/home/home.html')
```

Comment:
Also given how often the home page changes with regular A/B tests, plus another probably-imminent redesign, we'd only have to change it back to a view soon anyway. Best to leave it.